src/options/options.html

```diff
@@ -28,6 +28,23 @@ <h1>Omnibug Options</h1>
                 You can find the raw parameter names by hovering over the parameter name in Omnibug and it'll appear in parenthesis.</p>
         </fieldset>
 
+        <fieldset>
+            <div class="rxow">
+                <div class="col-label">Rename parameters</div>
+                <div class="col">
+                    <span id="renameParamStatus"></span>
+                    <textarea type="text" id="renameParam" placeholder="Rename Parameters"></textarea>
+                </div>
+            </div>
+            <p class="help">Specify JSON to change the name of your eVars, i.e.: 
```

Comment:
I think we'll need to specify here if it is the parameter key or the parameter name (e.g. v1 or eVar1)